clang/test/CAS/lit.local.cfg

```diff
@@ -1,3 +1,7 @@
 config.substitutions.append(('%{objroot}', config.clang_obj_root))
 if not config.have_ondisk_cas:
     config.unsupported = True
+
+import platform
+if platform.system() not in ['Linux', 'Darwin']:
```

Comment:
Maybe this should be `platform.system() == 'Windows'` since I'm guessing it works on other unix-like systems?

Reply:
Sure. That probably matches better with `LLVM_ON_UNIX` but neither option is an exact match.